Makefile

```diff
@@ -12,6 +12,8 @@ TOOLS_OUT := $(ROOT_OUT)/tools
 DOC_OUT := $(ROOT_OUT)/doc
 BUILD_VERSION ?=
 BUILD_TAG ?=
+DESTDIR ?=
```

Comment:
This var not used?

Reply:
It's being referred by install targets in Makefiles, such as:
...
./devicemodel/Makefile:	install -D --mode=0755 $(DM_OBJDIR)/$(PROGRAM) $(DESTDIR)/usr/bin/$(PROGRAM)
./devicemodel/Makefile:	install -D -t $(DESTDIR)/usr/share/acrn/samples/nuc $^
./devicemodel/Makefile:	install -D -t $(DESTDIR)/usr/share/acrn/samples/apl-mrb $^
./devicemodel/Makefile:	install -d $(DESTDIR)/usr/lib/systemd/system/
./devicemodel/Makefile:	install -p -D -m 0644 ./samples/apl-mrb/acrn_guest.service $(DESTDIR)/usr/lib/systemd/system
./devicemodel/Makefile:	install -d $(DESTDIR)/usr/share/acrn/bios
./devicemodel/Makefile:	install -D --mode=0664 -t $(DESTDIR)/usr/share/acrn/bios $^
...

here I just want give it a default empty value in the top Makefile, it's not a bug fix, but I think it would be easier to be read.

Reply:
`DESTDIR` should not be directly set in the `Makefile`, it is a built-in variable that is only used (and only useful) to the `(un)install` targets of a `Makefile` - see https://www.gnu.org/prep/standards/html_node/DESTDIR.html#DESTDIR

You're not setting it to any value but it's awkward to have it in the Makefile, it is expected that readers of Makefiles know this is a built-in variable. Can you remove it in your v2?

Reply:
You are absolutely right, will remove it.